src/lib/Scenes/Home/index.tsx

```diff
@@ -24,12 +26,21 @@ const TabBarContainer = styled.View`
 interface Props {
   selectedArtist?: string
   selectedTab?: number
+  tracking: any
```

Comment:
We should make the react-tracking typing for this prop use generics at some point so that it’s actually more useful than what you’re doing https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9e043c95dc87dd6e9c11abf746156876a9933edb/types/react-tracking/index.d.ts#L10

Reply:
:+1: